integreat_cms/api/urls.py

```diff
@@ -128,10 +129,21 @@
     path("<slug:region_slug>/", region_by_slug, name="region_by_slug"),
 ]
 
+socialmedia_api_urlpatterns = [
+    path("", socialmedia_headers, name="root_socialmedia_headers"),
+    path(
+        "<path:path>",
+        socialmedia_headers,
+        name="socialmedia_headers",
+    ),
+]
```

Comment:
I realized that it is not really simple to correctly match the pages here. 
`region_slug/language_slug` refers to a region, right. But what about `landing/de`? Due to the region middleware this would match and return 404. When using the other approach to match only specific slugs (as implemented here #1792) is not really flexible and requires action as soon as another path/slug joins the pool.
Therfore, I moved the matching/detection logic to `socialmedia_headers.py`.